.travis.yml

```diff
@@ -16,8 +16,15 @@ jobs:
   include:
     - stage: Tests
       name: Unit Tests
-      script: mvn clean test --batch-mode
-      after_success: mvn -DrepoToken=$COVERALLS_REPO_TOKEN org.jacoco:jacoco-maven-plugin:report org.eluder.coveralls:coveralls-maven-plugin:report
+      before_script:
+        - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+        - chmod +x ./cc-test-reporter
+        - ./cc-test-reporter before-build
+      script: mvn package --batch-mode
```

Comment:
just for my understanding: is there any reason for moving from test phase to package phase? It this required in order to run codeclimat scans?

Reply:
The `package` phase generates the JaCoCo report that is needed for code coverage import. 

Another way would be to add another line in the `after_script` which invokes `mvn org.jacoco:jacoco-maven-plugin:report`